config/locales/en/legislation.yml

```diff
@@ -64,6 +64,16 @@ en:
         no_open_processes: There aren't open processes
         no_next_processes: There aren't planned processes
         no_past_processes: There aren't past processes
+        section_header:
+          icon_alt: Legislation processes icon
+          title: Legislation processes
+          description: Participates in the debates and processes prior to the approval of a ordinance or a municipal action. Your opinion will be consider by the City Council.
+          help: Help about legislation processes
+        section_footer:
+          title: Help about legislation processes
+          help_text_1: "In participatory processes, the City Council offers to citizens the opportunity to participate in the drafting and modification of regulations affecting the city and to give its opinion on certain actions that it plans to carry out."
```

Comment:
`In the participatory processes, the City Council offers to citizens the opportunity to participate in the drafting and modification of regulations affecting the city and to give their opinion on certain actions planned to be accomplished.`